fdbserver/ApplyMetadataMutation.cpp

```diff
@@ -670,12 +670,12 @@ class ApplyMetadataMutationsImpl {
 
 				TraceEvent("CommitProxyInsertTenant", dbgid)
 				    .detail("Tenant", tenantName)
-				    .detail("Id", tenantEntry.id)
+				    .detail("Id", tenantEntry.tenantMinimalMetadata.id)
 				    .detail("Version", version);
 
-				(*tenantMap)[tenantEntry.id] = tenantName;
+				(*tenantMap)[tenantEntry.tenantMinimalMetadata.id].tenantName = tenantName;
 				if (tenantNameIndex) {
-					(*tenantNameIndex)[tenantName] = tenantEntry.id;
+					(*tenantNameIndex)[tenantName] = tenantEntry.tenantMinimalMetadata.id;
```

Comment:
I'm not sure that it matters much, but the privatized mutation we construct below could choose to just encode the minimal metadata if that's all the SS is going to use. I guess that would cost us something to serialize here but would then be a smaller mutation to handle and deserialize elsewhere.

Reply:
Yes, I was planning to look at minimizing metadata size at the end (after your work and mine has been merged) in a separate PR.